backend/clubs/views.py

```diff
@@ -4756,6 +4760,15 @@ def send_emails(self, *args, **kwargs):
                 and email_type == "acceptance"
             ):
                 template = acceptance_template
+
+                if send_invites and not dry_run:
```

Comment:
Do we need to check whether it's a dry run here? Seems like we're already doing it further down.

Reply:
I just didn't want to unnecessarily make invites that would otherwise never be sent

Reply:
Good catch. I didn't see that we were committing invites to the DB.

On that note: would it be better to create invites in bulk? After all, we know exactly which invites we're creating even before we go to the DB. Writing everything in one motion might be more efficient. 

This might be a good reference: https://docs.djangoproject.com/en/5.0/ref/models/querysets/#bulk-create

Reply:
Noted. Made changes

Reply:
Great. Also, I think after these changes, it's safe to remove the check for whether we're in a dry run or not.

Reply:
+1, you can remove the dry run check here